src/Mod/Spreadsheet/Gui/SheetTableView.cpp

```diff
@@ -73,34 +74,91 @@ bool SheetViewHeader::viewportEvent(QEvent *e) {
     return QHeaderView::viewportEvent(e);
 }
 
+static std::pair<int, int> selectedMinMaxRow(QModelIndexList list)
+{
+    int min = std::numeric_limits<int>::max();
+    int max = 0;
+    for (const auto & item : list) {
+        int row = item.row();
+        min = std::min(row, min);
+        max = std::max(row, max);
+    }
+    return std::pair<int, int>{min, max};
+}
+
+static std::pair<int, int> selectedMinMaxColumns(QModelIndexList list)
+{
+    int min = std::numeric_limits<int>::max();
+    int max = 0;
+    for (const auto & item : list) {
+        int column = item.column();
+        min = std::min(column, min);
+        max = std::max(column, max);
+    }
+    return std::pair<int, int>{min, max};
```

Comment:
Technically the `std::pair<int,int>` is no longer necessary, the compiler knows the return type and will construct it for you. That said, I don't object to its inclusion if you like the added clarity.